lib/routes/3kns/index.ts

```diff
@@ -0,0 +1,122 @@
+import { Data, DataItem, Route } from '@/types';
+import got from '@/utils/got';
+import { getCurrentPath } from '@/utils/helpers';
+import { parseDate } from '@/utils/parse-date';
+import { art } from '@/utils/render';
+import { load } from 'cheerio';
+import { Context } from 'hono';
+import * as path from 'node:path';
+const __dirname = getCurrentPath(import.meta.url);
+
+export const route: Route = {
+    path: '/:category?/:language?/:tag?/:pubDate?/:collection?/:order?',
+    categories: ['game'],
+    example: '/3kns',
+    parameters: {
+        category: '游戏类型，见下表',
+        language: '游戏语言，见下表',
+        tag: '游戏标签，见下表',
+        pubDate: '发售时间，见下表',
+        collection: '游戏集合，见下表',
+        order: '排序，按高分排序:desc;按低分排序:asc',
+    },
```

Comment:
Since there's quite a lot of parameters, it's suggested to use a single route parameter in [URLSearchParams style](https://github.com/DIYgod/RSSHub/blob/e8b0d7d2622ec43878e64706f8b89f6f1a2fee21/lib/routes/twitter/tweet.ts#L40) instead

Reply:
Or can I move these parameters to the search params, use `ctx.req.query` to get them?

Reply:
Not suggested since the cache will return exactly the same output even different search params.